Automattic/wp-calypso
# Fixes the Ads target for an advertising campaign

Related to SELFSERVE-630

## Proposed Changes

This PR fixes two things:
1- Changes the Ad target Operating System for Languages (the DSP doesn't support targeting OS right now)
2- Fixes an incorrect full page refresh when you navigate to the campaign details from a mobile view.

![Screenshot 2023-06-23 at 14 03 37](https://github.com/Automattic/wp-calypso/assets/1258162/620800e2-71b9-46dc-b1f9-d482384704dc)

![Screenshot 2023-06-23 at 14 04 12](https://github.com/Automattic/wp-calypso/assets/1258162/61617c0e-1e3a-4ae7-b47f-acbb149182cf)

## Testing Instructions

* Open the Live preview and Navigate to Tools->Advertising
* Click on the Campaigns navigation item
* Shrink the page to a mobile screen
* Click on any campaign
* Verify that the page didn't make a full page reload
* Verify that the new Target option "Languages" is displayed correctly.

## Pre-merge Checklist
